### PR TITLE
Task 2

### PR DESCRIPTION
**Pull request by jin-xiangwen**

### CODE REVIEW

File: ddl-10-11/CMakeLists.txt [hunk 1]
```diff
@@ -0,0 +1,17 @@
...
+
+project(main)
+
+set(CMAKE_CXX_STANDARD 11)
+set(CMAKE_CXX_STANDARD_REQUIRED ON)
+
+set(OpenCV_DIR /home/jin/下载/opencv-3.4.11/build)
+find_package(OpenCV REQUIRED)
+
+include_directories(/home/jin/下载/mindvision/include)
```
Comment (ZhengQiushi) [inline ddl-10-11/CMakeLists.txt:11]: do you install the SDK of mindvision on your ubuntu? Please tell me why you add this sentence.

File: ddl-10-11/main.cpp [hunk 2]
```diff
@@ -0,0 +1,71 @@
...
+    cv::Scalar scalarH = cv::Scalar(105, 255, 255);
+    inRange(HSV, scalarL, scalarH, binary);
+    imwrite("gray.jpg", binary);
+
+    //method 2: not using inRange
+    // double H=0,S=0,V=0;
+    // for(int i=0;i<HSV.rows;i++)
+    // {
+    //     for(int j=0;j<HSV.cols;j++)
+    //     {
```
Comment (ZhengQiushi) [inline ddl-10-11/main.cpp:39]: please remove all the meaningless comments in your code.

**Comment by jin-xiangwen** [comment]
1.直接按照群文件里的教程输入，就直接把话全都加进去了。还没有安装mindvision。

2.好的。会删除那些注释的。更改完后还要重新提交吗？





------------------&nbsp;原始邮件&nbsp;------------------
发件人:                                                                                                                        "JachinLin-TJ/2021RM-OPENCV"                                                                                    <notifications@github.com&gt;;
发送时间:&nbsp;2020年10月11日(星期天) 晚上8:52
收件人:&nbsp;"JachinLin-TJ/2021RM-OPENCV"<2021RM-OPENCV@noreply.github.com&gt;;
抄送:&nbsp;"金"<249195279@qq.com&gt;;"Author"<author@noreply.github.com&gt;;
主题:&nbsp;Re: [JachinLin-TJ/2021RM-OPENCV] Task 2 (#9)





 
@ZhengQiushi commented on this pull request.
 
 
In ddl-10-11/CMakeLists.txt:
 &gt; @@ -0,0 +1,17 @@ +cmake_minimum_required(VERSION 3.1) + +project(main) + +set(CMAKE_CXX_STANDARD 11) +set(CMAKE_CXX_STANDARD_REQUIRED ON) + +set(OpenCV_DIR /home/jin/下载/opencv-3.4.11/build) +find_package(OpenCV REQUIRED) + +include_directories(/home/jin/下载/mindvision/include)  
do you install the SDK of mindvision on your ubuntu? Please tell me why you add this sentence.
 
 
In ddl-10-11/main.cpp:
 &gt; +    /*Method 1 and method 2have no difference in the image output +when the parameters are the same. +  Using two methods to complete this is only for validation*/ + +    // method 1: using inRange +    cv::Scalar scalarL = cv::Scalar(100, 43, 250); +    cv::Scalar scalarH = cv::Scalar(105, 255, 255); +    inRange(HSV, scalarL, scalarH, binary); +    imwrite("gray.jpg", binary); + +    //method 2: not using inRange +    // double H=0,S=0,V=0; +    // for(int i=0;i<HSV.rows;i++) +    // { +    //     for(int j=0;j<HSV.cols;j++) +    //     {  
please remove all the meaningless comments in your code.

**Comment by ZhengQiushi** [comment]
please pay attention to all these details in your next task.



---Original---

**Pull request closed without merging**